test/ui-test/testSuites/suite_status/shared/scripts/sections/chat_names.py

```diff
@@ -2,14 +2,15 @@
 from sections.global_names import *
 
 # Main Window - chat related:
+navBarListView_Chat_navbar_StatusNavBarTabButton = {"checkable": True, "container": mainWindow_navBarListView_ListView, "objectName": "Chat-navbar", "type": "StatusNavBarTabButton", "visible": True}
```

Comment:
Any reason for not having cammel case in this `objectName`?

Reply:
That name is done programatically. It's the `name` or the section + `-navbar`

Reply:
Does this "Chat" name part depend on translation? If yes, maybe we should change this bc it will fail if the app is in another language and we can do something similar I did with setting items in this pr here: https://github.com/status-im/status-desktop/pull/7006/files#:~:text=object%20name%20helpers%3A-,_MAIN_MENU_ITEM_OBJ_NAME,-%3D%20%22%2DMainMenuItem%22 and `MenuPanel.qml` file.

Just mentioning it here but we can update it in a separate task if you prefere.